package.json

```diff
@@ -177,7 +177,7 @@
     "cross-env": "5.1.1",
     "css-loader": "0.28.7",
     "electron": "8.5.5",
-    "electron-builder": "22.9.1",
+    "electron-builder": "22.11.11",
```

Comment:
Whats up with this change? Have we verified this doesnt break anything on our builds/OTA updates

Reply:
See this https://github.com/CityOfZion/neon-wallet/pull/2227#issuecomment-913486861 for the reason. I don't know how to verify builds/OTA. I just saw a failing Windows build on the PR and tried to help by resolving it.